CRM/MembershipExtras/PaymentProcessor/OfflineRecurringContribution.php

```diff
@@ -63,6 +69,30 @@ public function get() {
     return $processor['values'][0];
   }
 
+  /**
+   * Returns the details of the default payment processor as per payment plan
+   * settings, or NULL if it does not exist.
+   *
+   * @return array
+   */
+  public function getDefaultProcessor() {
```

Comment:
I don't think method belongs to this class, it has nothing to do with. Maybe u can create a new class called CRM_MembershipExtras_Settings and move this method to it as a static method.